### PR TITLE
add changelog page

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-microej-sdk-6_add_changelog/SDK6UserGuide/sdk6ChangeLog.html

### CODE REVIEW

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 1]
```diff
@@ -0,0 +1,222 @@
+.. _sdk_6_changelog:
+
+SDK 6 Changelog
+---------------
+
+.. _changelog-0.9.0:
+
+[0.9.0] - Unreleased
+~~~~~~~~~~~~~~~~~~~~
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:9]: We should publish only released versions IMO, so either we remove this section for the moment, or we wait for the release of the version 0.9.0 to merge this doc.

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+Added
+"""""
+
+- Allow to depend on local Application project (microejApplication).
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:14]:
```suggestion
- Allow to depend on local Application project (dependency with ``microejApplication`` configuration).
```

File: SDK6UserGuide/sdk6MigrationNotes.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The build properties ``veePortDirs`` and ``veePortFiles`` have been merged into a single property ``veePortPaths``.
+To define a local VEE Port, set the build property ``veePortPaths`` in the ``microej`` configuration block 
+to the path of the VEE Port file (``.zip`` or ``.vde``) or to the source folder of the VEE Port ::
+
+   microej {
+     veePortPaths = listOf("C:\\path\\to\\my\\veePort\\file.zip")
+   }
+
+The ``veePortPaths`` property is defined as a list in order to provide multiple VEE Port files or source folders if it is needed ::
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6MigrationNotes.rst:23]:
```suggestion
The ``veePortPaths`` property is defined as a list in order to provide multiple VEE Port files or source folders if it is needed::
```

File: SDK6UserGuide/sdk6MigrationNotes.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+-------------------
+
+This section applies if MicroEJ SDK 6 ``0.9.0`` is used on a project that was created using MicroEJ SDK 6 ``0.8.0``.
+
+Merge of the veePortDirs and veePortFiles properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The build properties ``veePortDirs`` and ``veePortFiles`` have been merged into a single property ``veePortPaths``.
+To define a local VEE Port, set the build property ``veePortPaths`` in the ``microej`` configuration block 
+to the path of the VEE Port file (``.zip`` or ``.vde``) or to the source folder of the VEE Port ::
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6MigrationNotes.rst:17]:
```suggestion
to the path of the VEE Port file (``.zip`` or ``.vde``) or to the source folder of the VEE Port::
```

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 5]
```diff
@@ -0,0 +1,222 @@
+.. _sdk_6_changelog:
+
+SDK 6 Changelog
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:3]: I think we can remove "SDK 6".

File: SDK6UserGuide/sdk6ReleaseNotes.rst [hunk 6]
```diff
@@ -0,0 +1,12 @@
+.. _sdk_6_release_notes:
+
+Release Notes
+-------------
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ReleaseNotes.rst:4]: I agree to remove this page for the moment.

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: I moved the section of the version 0.9.0 in another branch so we can merge this one : https://github.com/MicroEJ/docs/tree/feature/microej-sdk-6_0.9.0

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**